bigbluebutton-html5/imports/ui/components/tabs/components/TabContentPpt/TabContentPptContainer.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import { withTracker } from 'meteor/react-meteor-data';
+import Service from '../../service';
+import PresentationService from '/imports/ui/components/presentation/presentation-uploader/service';
+import TabContentPpt from './TabContentPpt';
+
+const TabContentPptContainer = (props) => {
+  const options = [];
+  const allOptionPages = [];
+  let Slides = null;
+
+  const [selectedValue, setSelectedValue] = useState(null);
+
+  const handleChange = (e) => {
+    setSelectedValue(e.value);
+  };
+
+  const { presentations } = props;
```

Comment:
move this line to line no 8 below
`const TabContentPptContainer = (props) => {`

Reply:
fixed